pkg/node/volumemgr.go

```diff
@@ -983,7 +983,25 @@ func (m *VolumeManager) handleDriveStatusChange(ctx context.Context, drive *api.
 			}
 		}
 	}
-
+	lvgs, err := m.crHelper.GetLVGCRs(m.nodeID)
+	if err != nil {
+		ll.Errorf("Failed get LVG CRs, error: %v", err)
+	} else {
+		for _, lvg := range lvgs {
+			if util.ContainsString(lvg.Spec.Locations, drive.UUID) {
```

Comment:
I think that we can stop loop after we find first LVG because, as far as I know, drive can belong to single LVG only. Also, we already have similar function in CR helper https://github.com/dell/csi-baremetal/blob/5642af2ef8a9d68ddad061227d4c4667fe097e22/pkg/base/k8s/cr_helper.go#L151 Maybe we can make the function public and use it here? Here we can leave only status update for LVG.

Reply:
Done